CHANGELOG.md

```diff
@@ -7,11 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v1.8.0-0.27b0...HEAD)
 
+- `opentelemetry-instrumentation-django` Django: Conditionally create SERVER spans
```

Comment:
Please fix the CHANGELOG. Remove the duplicate previous release entry and move your entry to fixed list below.

<img width="1049" alt="Screenshot 2022-01-06 at 6 58 10 PM" src="https://user-images.githubusercontent.com/22846633/148390049-5d78eb80-77a5-4c97-901b-d83ef1086e95.png">